kit/inputs/src/useApplyMask/amount/onChangeAmount.ts

```diff
@@ -0,0 +1,38 @@
+import { runOnUI, SharedValue } from 'react-native-reanimated';
+import { uiLocalized } from '@tonlabs/localization';
+import { runUIOnChangeAmount } from './runUIOnChangeAmount';
+import type { ChangeText, MoveCarret } from '../../UIMaterialTextView/types';
+
+export function onChangeAmount(
+    rawNumber: string,
+    selectionStart: SharedValue<number>,
+    selectionEnd: SharedValue<number>,
+    lastNormalizedText: SharedValue<string>,
+    lastText: SharedValue<string>,
+    changeText: ChangeText,
+    moveCarret: MoveCarret,
+    skipNextOnSelectionChange: SharedValue<boolean>,
+) {
+    const {
+        grouping: integerSeparator,
+        decimal: delimeter,
+        decimalGrouping: fractionalSeparator,
+    } = uiLocalized.localeInfo.numbers;
+
+    const inputSeparators = [',', '.', '/', 'б', 'ю'];
```

Comment:
this Cyrillic symbols scares me... (I don't like it here for sure!)
You can take all the required data from `localizedService.localeInfo`

Reply:
I've added new `decimalAlternative` to `uiLocalized.localeInfo.numbers`.